libshaderc/include/shaderc.h

```diff
@@ -63,6 +63,15 @@ typedef enum {
   shaderc_profile_es,
 } shaderc_profile;
 
+// Used in result module (shaderc_spv_module) to tell the type of errors
+// generated during compilation.
+typedef enum {
+  shaderc_none_error,
+  shaderc_shader_kind_error,  // errors due to failure in shader stage
+                              // deduction.
+  shaderc_compilation_error,
+} shaderc_error_type;
+
 // Usage examples:
 //
```

Comment:
Might be best to re-name the enum, 
shaderc_none_error makes me sad.

typedef enum {
   shaderc_compilation_result_success,
   shaderc_compilation_result_failed_invalid_stage,
   shaderc_compilation_result_compilation_failed
} shaderc_compilation_result;